src/Couchbase.Lite.Shared/API/Query/SelectResult.cs

```diff
@@ -44,7 +52,7 @@ public static class SelectResult
         /// <returns>The instantiated instance</returns>
         /// <remarks>Equivalent to <c>SelectResult.Expression(Expression.Property(property))</c></remarks>
         [NotNull]
-        public static ISelectResultAs Property(string property) => new QuerySelectResult(Lite.Query.Expression.Property(property));
+        public static ISelectResultAs Property([NotNull]string property) => new QuerySelectResult(Lite.Query.Expression.Property(property));
```

Comment:
Missing `CBDebug` check

Reply:
Lite.Query.Expression.Property([NotNull]string property) already check for property string to be null and throw exception if it's null. That's why I did not add a CBDebug here. Please let me know if you still want a CBDebug here.

Reply:
If you know it only calls to another method which already has a CBDebug then you don't need to.